Peereval visibility bugs

sr2072 opened the pull request:
- fixed Question Builder moving off the screen
- Mobile Friendly peer evaluation forms for student users added by Brendan

sauravr09 committed:
Fix QuestionBuilder moving off the screen

sauravr09 committed:
peer eval for student users made mobile friendly by Brendan

tjp4618 committed:
changed it so the modal stays stationary in the middle of the screen when adding a new question to the questionbuilder

tjp4618 commented:
What you had worked, but instead of changing the style of the whole modal, I just changed the maxHeight of the grid column in the modal so that it doesn't mess with the position of the modal. It should look better now!

sauravr09 committed:
center question builder modal when it is opened.

sr2072 commented:
I can understand the new changes, but the original issue of the Question Builder Modal moving off the screen when opened at a normal zoom of 100% or higher is back. This issue presents itself when the Modal is given additional style trying to center it (especially transform: translate()). From my understanding, the Modal from semantic UI seems to already be centered by default and the additional styling is not needed. 

I have removed the additional centering styles from modal, but kept 'position: fixed'. The QuestionBuilder modal should be centered when it is opened now! Feel free to double check and it should be ready for merge!

tjp4618 commented:
I see what you mean and yeah when I zoomed in the same issue occurred where it went off the screen. The main reason I put style back was because without it, the modal was moving to the bottom right edge of the page. I attached two pictures: one that shows what happens when I open the question builder on normal zoom, and the other one when I zoom in.

<img width="1680" alt="Screenshot 2025-03-06 at 4 50 15 PM" src="https://github.com/user-attachments/assets/a61311f3-95d2-4375-8691-7f1c2760aba2" />
<img width="1680" alt="Screenshot 2025-03-06 at 4 50 27 PM" src="https://github.com/user-attachments/assets/5daabba2-262f-4d6b-84e2-45d7e4c78817" />

It centers itself when I zoom in but when I am at 100%, it stays in the bottom right. I'm not sure if this issue happens with you but I can try to figure out why it's doing that on my end.

tjp4618 committed:
added style that centers modal even when zooming in

tjp4618 commented:
You were correct that it was the previous styling that was messing with the modal when zooming in, so I fixed the styling so that it should work and be centered. Let me know if it works properly!

sr2072 commented:
Great, thanks! I just pulled the changes and tested it out. Everything looks good and ready to go now!

bdh5060 commented:
looks good to me
